tests/test_pilot_airfoil.py

```diff
@@ -28,7 +28,7 @@
 import unittest
 
 import modmesh as mm
-from modmesh.pilot import airfoil
+from modmesh.pilot.airfoil import _naca
```

Comment:
Be specific to what sub-module to test.  Importing private modules from outside the package should only be allowed in testing.